task1/main.c

```diff
@@ -0,0 +1,113 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+
+bool isDigit(char symbol)
+{
+    return symbol >= 48 && symbol <= 57;
```

Comment:
```suggestion
    return symbol >= '0' && symbol <= '9';
```
Так нагляднее будет.